static/app/views/ddm/ddm.tsx

```diff
@@ -37,7 +46,9 @@ function DDM() {
             </Layout.HeaderContent>
             <Layout.HeaderActions>
               <ButtonBar gap={1}>
-                <FeatureFeedback featureName="DDM" buttonProps={{size: 'sm'}} />
+                <GithubFeedbackButton href="https://github.com/getsentry/sentry/discussions/54956">
+                  Give Feedback
```

Comment:
Let's translate this or even better, make it a default text of the GithubFeedbackButton so that you don't have to think about this unless you want something more customized.